lib/bitreserve/api/contact.rb

```diff
@@ -0,0 +1,34 @@
+module Bitreserve
+  module API
+    module Contact
+      def all_contacts
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS,
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_objects(:get, request_data)
+      end
+
+      def find_contact(id: nil)
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS + "/#{id}",
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_object(:get, request_data)
+      end
+
+      def create_contact(**args)
```

Comment:
What about having keyword arguments for all the possibilities? Do you think it's too limitative?